src/features/onboarding/model/types.ts

```
@@ -11,6 +11,25 @@ export type OnBoardingFormData = {
+export type OnboardingInitData = {
+  nickname: string | null;
+  email: string | null;
+  profileImageUrl: string | null;
+};
+
+export type OnboardingState = OnboardingInitData & {
+  setOnboarding: (auth: OnboardingInitData) => void;
+  clearOnboarding: () => void;
+};
+
+export type TrackPart =
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TrackPart'. Consider to delete it or rename it.

'TrackPart' is defined here:


(lint/suspicious/noRedeclare)